docs/userguide/008_The_Library_API.adoc

```diff
@@ -318,6 +318,32 @@ You can configure the location of the violation store within `archunit.propertie
 freeze.store.default.path=/some/path/in/a/vcs/repo
 ----
 
+Furthermore, it is possible to configure
+
+[source,options="nowrap"]
+.archunit.properties
+----
+# must be set to true to allow the creation of a new violation store
+# default is false
+freeze.store.default.allowStoreCreation=true
+
+# can be set to false to forbid updates of the violations stored for frozen rules
+# default is true
+freeze.store.default.allowStoreUpdate=false
+----
+
+This can help in CI environments to prevent a misconfiguration.
+For example, if a CI job should never need to update the violation store, it can be prevented.
+Also a misconfiguration of the store path in a CI environment can be prevented by forbidding the creation of a new violation store.
```

Comment:
Maybe simpler:
```
This can help in CI environments to prevent a misconfiguration:
For example, a CI job should probably never update the violation store nor create a new one.
```

Reply:
I don't know, if "a CI build should probably never update" hold true, because in all the contexts where I've introduced `FreezingArchRule` so far, the CI was always updating the `ViolationStore` to automatically reduce the number of violations.
But we could just drop the update part and only mention store creation. Because that is IMHO always fishy for typical CI runs.